lib/utils/oclif-utils.ts

```diff
@@ -26,9 +26,9 @@ export class CommandHelp {
 	constructor(public command: { args?: any[] }) {}
 
 	protected arg(arg: Config.Command['args'][0]): string {
-		const name = arg.name.toUpperCase();
+		const name = arg.name;
 		if (arg.required) {
-			return `${name}`;
+			return `<${name}>`;
 		}
 		return `[${name}]`;
 	}
```

Comment:
These changes required to preserve camelCase in usages.

Reply:
This would be a departure from oclif's framework default style:
https://github.com/oclif/plugin-help/blob/v2.2.3/src/command.ts#L115-L117

I understand that oclif intentionally uses all upper case, instead of angle brackets, as a visual indication of a value that should be replaced by the user in actual usage. It is a framework style where oclif differs from Capitano. This change is also affecting the oclif style already in use in documentation (`cli.markdown`) for previously converted CLI commands like `os configure`, `env add`, etc.

The `capitanoizeOclifUsage` function below, as the name suggests, attempts to re-format oclif's style to match Capitano, but it is used specifically for the `balena help` output that presents the command usage as one command per line, where the difference between oclif and Capitano would be too obvious / too hard on the eye.

Reply:
@pdcastro I guess we need make a decision on what the format should be across the board.  

Personally I prefer the capitano style.  The oclif style looks a bit ugly and jarring to my eyes (any muti-word param names become quite unreadable when all caps).

The capitanoizeOclifUsage function did not preserve camelCase names though, so made multi-word names hard to read whenlikethis, hence these changes.  I think we hadn't noticed this before, because there were no multi-word parameter names in any of oclif commands converted so far.

Reply:
We also have the inconsistency of how oclif shows it's options in help with `=`, e.g. `--application=application` (a format we don't use in any of our usage examples or blog post etc.)